docs/hooks.md

```diff
@@ -51,4 +46,49 @@ Ark v0.7.0+ continues to support the original (deprecated) way to specify pre ho
 Please see the documentation on the [Backup API Type][1] for how to specify hooks in the Backup
 spec.
 
+## Hook Example with fsfreeze
+
+We are going to walk through using both pre and post hooks for freezing a file system. Freezing the
+file system is useful to ensure that all pending disk I/O operations have completed prior to taking a snapshot.
+
+We will be using [example/nginx-app/with-pv.yaml][2] for this example. Follow the [steps for your provider][3] to
+setup this example.
+
+### Annotations
+
+The Ark [example/nginx-app/with-pv.yaml][2] serves as an example of adding the pre and post hook annotations directly
+to your declarative deployment. Below is an example of what updating an object in place might look like.
+
+Place Ark in restore only mode. This will prevent Ark backups from running while you are adding the
+annotations and avoid the condition where the pre hook freezes the file system, but there is no
+post hook setup to unfreeze it.
+
+```shell
+kubectl patch -n heptio-ark config default --type merge -p '{"restoreOnlyMode": true}'
+```
+
+Now you patch your pod with the required annotations.
+
+```shell
+kubectl annotate pod -n nginx-example -l app=nginx pre.hook.backup.ark.heptio.com/command='["/sbin/fsfreeze", "--freeze", "/var/log/nginx"]'
```

Comment:
You *could* apply all of these annotations in a single `kubectl` call, e.g. `kubectl annotate pod a=b c=d ...` -- which I think would remove the need to flip ark into restore-only mode and back.  I think that'd be preferable since although I get the restore-only mode thing, it distracts a little from the main point